software/uf2_build/build_uf2.bat

```diff
@@ -0,0 +1,2 @@
+docker build --tag europi_buildenv --file software/uf2_build/Dockerfile .
+docker run --rm -it -v %cd%:/europi europi_buildenv
```

Comment:
[question] Doesn't the `-it` make this docker execution interactive? I guess that when the bat script exits then it closes the interactive session. So it probably doesn't hurt anything, but it's also unnecessary.